packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -317,7 +305,7 @@ class _CpuProfilerState extends State<CpuProfiler>
     );
     // TODO(kenz): make this order configurable.
     return [
-      if (widget.summaryView != null) widget.summaryView,
+      if (widget.summaryView != null) widget.summaryView!,
```

Comment:
We can remove the redundant null check by putting `widget.summaryView` into a local before this return.

Reply:
done